app/i18n/nl/admin.php

```diff
@@ -112,6 +112,13 @@
 		),
 		'title' => 'Uitbreidingen',
 		'user' => 'Gebruikersuitbreidingen',
+		'community' => 'Gebruikersuitbreidingen beschikbaar voor download',
```

Comment:
With the new shorter version this is now `Gebruikersuitbreidingen beschikbaar` or `Beschikbare gebruikersuitbreidingen `:-)